--- config-2_1.yml	2023-02-02 21:24:39.000000000 -0500
+++ config-2_1.yml.MIDRES	2023-02-02 21:25:05.000000000 -0500
@@ -157,14 +157,14 @@
 j8_par_executor: &j8_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j8_small_par_executor: &j8_small_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j8_small_executor: &j8_small_executor
   executor:
@@ -172,29 +172,41 @@
     exec_resource_class: medium
   parallelism: 1
 
+j8_large_par_executor: &j8_large_par_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: large
+  parallelism: 50
+
+j8_very_large_par_executor: &j8_very_large_par_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: xlarge
+  parallelism: 100
+
 j8_medium_par_executor: &j8_medium_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 4
 
 j8_seq_executor: &j8_seq_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j11_par_executor: &j11_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_small_par_executor: &j11_small_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j11_small_executor: &j11_small_executor
   executor:
@@ -205,44 +217,56 @@
 j11_medium_par_executor: &j11_medium_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 4
+
+j11_large_par_executor: &j11_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 50
 
 j11_seq_executor: &j11_seq_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j8_repeated_utest_executor: &j8_repeated_utest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j8_repeated_dtest_executor: &j8_repeated_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j8_repeated_upgrade_dtest_executor: &j8_repeated_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 25
 
 j8_repeated_jvm_upgrade_dtest_executor: &j8_repeated_jvm_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j11_repeated_utest_executor: &j11_repeated_utest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_repeated_dtest_executor: &j11_repeated_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j8_separate_jobs: &j8_separate_jobs
   jobs:
@@ -1819,7 +1843,7 @@
           target: testclasslist-system-keyspace-directory
 
   j8_dtests_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1833,7 +1857,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j8_dtests_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1847,7 +1871,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j11_dtests_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1862,7 +1886,7 @@
         pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j11_dtests_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1877,7 +1901,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j8_dtests:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1891,7 +1915,7 @@
           pytest_extra_args: '--skip-resource-intensive-tests'
 
   j11_dtests:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1906,7 +1930,7 @@
         pytest_extra_args: '--skip-resource-intensive-tests'
 
   j8_upgrade_dtests:
-    <<: *j8_par_executor
+    <<: *j8_very_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1920,7 +1944,7 @@
           pytest_extra_args: '--execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all'
 
   j8_cqlsh_dtests_py3_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1935,7 +1959,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py3_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1950,7 +1974,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py38_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1968,7 +1992,7 @@
           python_version: '3.8'
 
   j8_cqlsh_dtests_py311_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1986,7 +2010,7 @@
           python_version: '3.11'
 
   j8_cqlsh_dtests_py38_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2004,7 +2028,7 @@
           python_version: '3.8'
 
   j8_cqlsh_dtests_py311_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2022,7 +2046,7 @@
           python_version: '3.11'
 
   j8_cqlsh_dtests_py3:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2037,7 +2061,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py38:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2055,7 +2079,7 @@
           python_version: '3.8'
 
   j8_cqlsh_dtests_py311:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2073,7 +2097,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py3_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2088,7 +2112,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py3_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2103,7 +2127,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2121,7 +2145,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2139,7 +2163,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py38_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2157,7 +2181,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2175,7 +2199,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py3:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2190,7 +2214,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2208,7 +2232,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2226,7 +2250,7 @@
           python_version: '3.11'
 
   j8_dtests_large_vnode:
-    <<: *j8_par_executor
+    <<: *j8_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2240,7 +2264,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j8_dtests_large:
-    <<: *j8_par_executor
+    <<: *j8_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2254,7 +2278,7 @@
           pytest_extra_args: '--only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large_vnode:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2268,7 +2292,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
