--- config_template_11_and_17.yml	2023-06-13 12:42:25.266537877 +0100
+++ config_template_11_and_17.yml.PAID	2023-06-13 12:50:31.077931044 +0100
@@ -141,14 +141,14 @@
 j11_par_executor: &j11_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_small_par_executor: &j11_small_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j11_small_executor: &j11_small_executor
   executor:
@@ -156,30 +156,41 @@
     exec_resource_class: medium
   parallelism: 1
 
+j11_large_par_executor: &j11_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 50
+
+j11_very_large_par_executor: &j11_very_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 100
+
 j11_medium_par_executor: &j11_medium_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 4
 
 j11_seq_executor: &j11_seq_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
     exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j17_par_executor: &j17_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j17_small_par_executor: &j17_small_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j17_small_executor: &j17_small_executor
   executor:
@@ -190,34 +201,44 @@
 j17_medium_par_executor: &j17_medium_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 4
+
+j17_large_par_executor: &j17_large_par_executor
+  executor:
+    name: java17-executor
+    exec_resource_class: large
+  parallelism: 50
 
 j17_seq_executor: &j17_seq_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j11_repeated_utest_executor: &j11_repeated_utest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_repeated_dtest_executor: &j11_repeated_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j17_repeated_utest_executor: &j17_repeated_utest_executor
   executor:
     name: java17-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j17_repeated_dtest_executor: &j17_repeated_dtest_executor
   executor:
     name: java17-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j11_separate_jobs: &j11_separate_jobs
   jobs:
@@ -1883,7 +1904,7 @@
           target: testclasslist-system-keyspace-directory
 
   j11_dtests_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1897,7 +1918,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j11_dtests_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1911,7 +1932,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j17_dtests_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1926,7 +1947,7 @@
         pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j17_dtests_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1941,7 +1962,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j11_dtests:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1955,7 +1976,7 @@
           pytest_extra_args: '--skip-resource-intensive-tests'
 
   j17_dtests:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1970,7 +1991,7 @@
         pytest_extra_args: '--skip-resource-intensive-tests'
 
   j11_cqlsh_dtests_py3_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1985,7 +2006,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py3_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2000,7 +2021,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2018,7 +2039,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2036,7 +2057,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py38_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2054,7 +2075,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2072,7 +2093,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py3:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2087,7 +2108,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2105,7 +2126,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2123,7 +2144,7 @@
           python_version: '3.11'
 
   j17_cqlsh_dtests_py3_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2138,7 +2159,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j17_cqlsh_dtests_py3_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2153,7 +2174,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j17_cqlsh_dtests_py38_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2171,7 +2192,7 @@
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2189,7 +2210,7 @@
           python_version: '3.11'
 
   j17_cqlsh_dtests_py38_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2207,7 +2228,7 @@
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2225,7 +2246,7 @@
           python_version: '3.11'
 
   j17_cqlsh_dtests_py3:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2240,7 +2261,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j17_cqlsh_dtests_py38:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2258,7 +2279,7 @@
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2276,7 +2297,7 @@
           python_version: '3.11'
 
   j11_dtests_large_vnode:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2290,7 +2311,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2304,7 +2325,7 @@
           pytest_extra_args: '--only-resource-intensive-tests --force-resource-intensive-tests'
 
   j17_dtests_large_vnode:
-    <<: *j17_par_executor
+    <<: *j17_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2318,7 +2339,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j17_dtests_large:
-    <<: *j17_par_executor
+    <<: *j17_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
