--- .circleci/config-2_1.yml	2021-10-18 12:30:30.000000000 +0100
+++ .circleci/config-2_1.yml.HIGHRES	2021-10-18 15:22:24.000000000 +0100
@@ -101,14 +101,14 @@
 j8_par_executor: &j8_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_small_par_executor: &j8_small_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 5
 
 j8_small_executor: &j8_small_executor
   executor:
@@ -119,62 +119,68 @@
 j8_medium_par_executor: &j8_medium_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 2
 
 j8_seq_executor: &j8_seq_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: xlarge
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j11_par_executor: &j11_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
-j11_small_par_executor: &j11_small_par_executor
+j11_small_executor: &j11_small_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1
 
-j11_small_executor: &j11_small_executor
+j11_small_par_executor: &j11_small_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: medium
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 2
 
 j8_repeated_utest_executor: &j8_repeated_utest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_repeated_dtest_executor: &j8_repeated_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_repeated_upgrade_dtest_executor: &j8_repeated_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_repeated_jvm_upgrade_dtest_executor: &j8_repeated_jvm_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j11_repeated_utest_executor: &j11_repeated_utest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j11_repeated_dtest_executor: &j11_repeated_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_with_dtests_jobs: &j8_with_dtests_jobs
   jobs:
