--- a/config_template_11_and_17.yml
+++ b/config_template_11_and_17.yml.PAID
@@ -141,14 +141,14 @@ default_env_vars: &default_env_vars
 j11_par_executor: &j11_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_small_par_executor: &j11_small_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j11_small_executor: &j11_small_executor
   executor:
@@ -156,30 +156,41 @@ j11_small_executor: &j11_small_executor
     exec_resource_class: medium
   parallelism: 1
 
+j11_large_par_executor: &j11_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 50
+
+j11_very_large_par_executor: &j11_very_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 100
+
 j11_medium_par_executor: &j11_medium_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 4
 
 j11_seq_executor: &j11_seq_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
     exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j17_par_executor: &j17_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j17_small_par_executor: &j17_small_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j17_small_executor: &j17_small_executor
   executor:
@@ -190,34 +201,44 @@ j17_small_executor: &j17_small_executor
 j17_medium_par_executor: &j17_medium_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 4
+
+j17_large_par_executor: &j17_large_par_executor
+  executor:
+    name: java17-executor
+    exec_resource_class: large
+  parallelism: 50
 
 j17_seq_executor: &j17_seq_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j11_repeated_utest_executor: &j11_repeated_utest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_repeated_dtest_executor: &j11_repeated_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j17_repeated_utest_executor: &j17_repeated_utest_executor
   executor:
     name: java17-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j17_repeated_dtest_executor: &j17_repeated_dtest_executor
   executor:
     name: java17-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j11_separate_jobs: &j11_separate_jobs
   jobs:
@@ -1877,7 +1898,7 @@ jobs:
           target: testclasslist-system-keyspace-directory
 
   j11_dtests_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1891,7 +1912,7 @@ jobs:
           pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j11_dtests_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1905,7 +1926,7 @@ jobs:
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j17_dtests_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1920,7 +1941,7 @@ jobs:
         pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j17_dtests_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1935,7 +1956,7 @@ jobs:
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j11_dtests:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1949,7 +1970,7 @@ jobs:
           pytest_extra_args: '--skip-resource-intensive-tests'
 
   j17_dtests:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1964,7 +1985,7 @@ jobs:
         pytest_extra_args: '--skip-resource-intensive-tests'
 
   j11_cqlsh_dtests_py3_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1979,7 +2000,7 @@ jobs:
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py3_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1994,7 +2015,7 @@ jobs:
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2012,7 +2033,7 @@ jobs:
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2030,7 +2051,7 @@ jobs:
           python_version: '3.11'
 
   j11_cqlsh_dtests_py38_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2048,7 +2069,7 @@ jobs:
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2066,7 +2087,7 @@ jobs:
           python_version: '3.11'
 
   j11_cqlsh_dtests_py3:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2081,7 +2102,7 @@ jobs:
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2099,7 +2120,7 @@ jobs:
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2117,7 +2138,7 @@ jobs:
           python_version: '3.11'
 
   j17_cqlsh_dtests_py3_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2132,7 +2153,7 @@ jobs:
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j17_cqlsh_dtests_py3_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2147,7 +2168,7 @@ jobs:
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j17_cqlsh_dtests_py38_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2165,7 +2186,7 @@ jobs:
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2183,7 +2204,7 @@ jobs:
           python_version: '3.11'
 
   j17_cqlsh_dtests_py38_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2201,7 +2222,7 @@ jobs:
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2219,7 +2240,7 @@ jobs:
           python_version: '3.11'
 
   j17_cqlsh_dtests_py3:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2234,7 +2255,7 @@ jobs:
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j17_cqlsh_dtests_py38:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2252,7 +2273,7 @@ jobs:
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2270,7 +2291,7 @@ jobs:
           python_version: '3.11'
 
   j11_dtests_large_vnode:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2284,7 +2305,7 @@ jobs:
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2298,7 +2319,7 @@ jobs:
           pytest_extra_args: '--only-resource-intensive-tests --force-resource-intensive-tests'
 
   j17_dtests_large_vnode:
-    <<: *j17_par_executor
+    <<: *j17_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2312,7 +2333,7 @@ jobs:
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j17_dtests_large:
-    <<: *j17_par_executor
+    <<: *j17_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
