--- config-2_1.yml	2022-11-15 12:30:50.000000000 -0500
+++ config-2_1.yml.MIDRES	2022-11-15 12:35:46.000000000 -0500
@@ -157,14 +157,14 @@
 j8_par_executor: &j8_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j8_small_par_executor: &j8_small_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j8_small_executor: &j8_small_executor
   executor:
@@ -172,29 +172,41 @@
     exec_resource_class: medium
   parallelism: 1
 
+j8_large_par_executor: &j8_large_par_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: large
+  parallelism: 50
+
+j8_very_large_par_executor: &j8_very_large_par_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: xlarge
+  parallelism: 100
+
 j8_medium_par_executor: &j8_medium_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 4
 
 j8_seq_executor: &j8_seq_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j11_par_executor: &j11_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_small_par_executor: &j11_small_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j11_small_executor: &j11_small_executor
   executor:
@@ -205,44 +217,56 @@
 j11_medium_par_executor: &j11_medium_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 4
+
+j11_large_par_executor: &j11_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 50
 
 j11_seq_executor: &j11_seq_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j8_repeated_utest_executor: &j8_repeated_utest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j8_repeated_dtest_executor: &j8_repeated_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j8_repeated_upgrade_dtest_executor: &j8_repeated_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 25
 
 j8_repeated_jvm_upgrade_dtest_executor: &j8_repeated_jvm_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j11_repeated_utest_executor: &j11_repeated_utest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_repeated_dtest_executor: &j11_repeated_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j8_separate_jobs: &j8_separate_jobs
   jobs:
@@ -1771,7 +1795,7 @@
           target: testclasslist-system-keyspace-directory
 
   j8_dtests_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1785,7 +1809,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j8_dtests_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1799,7 +1823,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j11_dtests_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1814,7 +1838,7 @@
         pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j11_dtests_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1829,7 +1853,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j8_dtests:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1843,7 +1867,7 @@
           pytest_extra_args: '--skip-resource-intensive-tests'
 
   j11_dtests:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1858,7 +1882,7 @@
         pytest_extra_args: '--skip-resource-intensive-tests'
 
   j8_upgrade_dtests:
-    <<: *j8_par_executor
+    <<: *j8_very_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1872,7 +1896,7 @@
           pytest_extra_args: '--execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all'
 
   j8_cqlsh_dtests_py3_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1887,7 +1911,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py3_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1902,7 +1926,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py38_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1920,7 +1944,7 @@
           python_version: '3.8'
 
   j8_cqlsh_dtests_py38_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1938,7 +1962,7 @@
           python_version: '3.8'
 
   j8_cqlsh_dtests_py3:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1953,7 +1977,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py38:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1971,7 +1995,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py3_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1986,7 +2010,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py3_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2001,7 +2025,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2019,7 +2043,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py38_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2037,7 +2061,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py3:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2052,7 +2076,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2070,7 +2094,7 @@
           python_version: '3.8'
 
   j8_dtests_large_vnode:
-    <<: *j8_par_executor
+    <<: *j8_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2084,7 +2108,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j8_dtests_large:
-    <<: *j8_par_executor
+    <<: *j8_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2098,7 +2122,7 @@
           pytest_extra_args: '--only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large_vnode:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2112,7 +2136,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
