--- config-2_1.yml	2023-02-02 15:43:38.000000000 -0500
+++ config-2_1.yml.HIGHRES	2023-02-02 16:29:38.000000000 -0500
@@ -40,8 +40,8 @@
     CASSANDRA_SKIP_SYNC: true
     DTEST_REPO: https://github.com/apache/cassandra-dtest.git
     DTEST_BRANCH: trunk
-    CCM_MAX_HEAP_SIZE: 1024M
-    CCM_HEAP_NEWSIZE: 256M
+    CCM_MAX_HEAP_SIZE: 2048M
+    CCM_HEAP_NEWSIZE: 512M
 
     # Whether the repeated test iterations should stop on the first failure by default.
     REPEATED_TESTS_STOP_ON_FAILURE: false
@@ -127,14 +127,14 @@
 j8_par_executor: &j8_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_small_par_executor: &j8_small_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 2
 
 j8_small_executor: &j8_small_executor
   executor:
@@ -145,34 +145,38 @@
 j8_medium_par_executor: &j8_medium_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 2
 
 j8_seq_executor: &j8_seq_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: xlarge
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j8_repeated_utest_executor: &j8_repeated_utest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_repeated_dtest_executor: &j8_repeated_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_repeated_upgrade_dtest_executor: &j8_repeated_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 j8_repeated_jvm_upgrade_dtest_executor: &j8_repeated_jvm_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 100
 
 separate_jobs: &separate_jobs
   jobs:
@@ -1189,7 +1193,7 @@
             # Put manually specified tests and automatically detected tests together, removing duplicates
             tests=$(echo <<parameters.tests>> | sed -e "s/<nil>//" | sed -e "s/ //" | tr "," "\n" | tr " " "\n" | sort -n | uniq -u)
             echo "Tests to be repeated: ${tests}"
-            
+
             # Prepare the testtag for the target, used by the test macro in build.xml to group the output files
             target=<<parameters.target>>
             testtag=""
