--- config_template.yml	2024-01-05 00:32:24.148600479 +0000
+++ config_template.yml.PAID	2024-01-05 00:32:40.861079981 +0000
@@ -157,20 +157,20 @@
 j11_par_executor: &j11_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_small_par_executor: &j11_small_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: medium
+  parallelism: 10
 
 j11_small_parallel_executor: &j11_small_parallel_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: medium
+  parallelism: 4
 
 j11_small_executor: &j11_small_executor
   executor:
@@ -178,30 +178,41 @@
     exec_resource_class: medium
   parallelism: 1
 
+j11_large_par_executor: &j11_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 50
+
+j11_very_large_par_executor: &j11_very_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 100
+
 j11_medium_par_executor: &j11_medium_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 4
 
 j11_seq_executor: &j11_seq_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
     exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j17_par_executor: &j17_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j17_small_par_executor: &j17_small_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: medium
+  parallelism: 10
 
 j17_small_executor: &j17_small_executor
   executor:
@@ -212,44 +223,56 @@
 j17_medium_par_executor: &j17_medium_par_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: xlarge
+  parallelism: 4
+
+j17_large_par_executor: &j17_large_par_executor
+  executor:
+    name: java17-executor
+    exec_resource_class: large
+  parallelism: 50
 
 j17_seq_executor: &j17_seq_executor
   executor:
     name: java17-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j11_repeated_jvm_upgrade_dtest_executor: &j11_repeated_jvm_upgrade_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_repeated_utest_executor: &j11_repeated_utest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_repeated_dtest_executor: &j11_repeated_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j17_repeated_utest_executor: &j17_repeated_utest_executor
   executor:
     name: java17-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j17_repeated_dtest_executor: &j17_repeated_dtest_executor
   executor:
     name: java17-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j11_repeated_upgrade_dtest_executor: &j11_repeated_upgrade_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 25
 
 j11_separate_jobs: &j11_separate_jobs
   jobs:
@@ -1920,7 +1943,7 @@
           target: testclasslist-system-keyspace-directory
 
   j11_dtests_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1934,7 +1957,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j11_dtests_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1948,7 +1971,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j17_dtests_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1963,7 +1986,7 @@
         pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j17_dtests_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1978,7 +2001,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j11_dtests:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1992,7 +2015,7 @@
           pytest_extra_args: '--skip-resource-intensive-tests'
 
   j17_dtests:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -2007,7 +2030,7 @@
         pytest_extra_args: '--skip-resource-intensive-tests'
 
   j11_upgrade_dtests:
-    <<: *j11_par_executor
+    <<: *j11_very_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -2021,7 +2044,7 @@
         pytest_extra_args: '--execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all'
 
   j11_cqlsh_dtests_py38_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2039,7 +2062,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2057,7 +2080,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py38_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2075,7 +2098,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2093,7 +2116,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py38:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2111,7 +2134,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2129,7 +2152,7 @@
           python_version: '3.11'
 
   j17_cqlsh_dtests_py38_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2147,7 +2170,7 @@
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311_vnode:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2165,7 +2188,7 @@
           python_version: '3.11'
 
   j17_cqlsh_dtests_py38_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2183,7 +2206,7 @@
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311_offheap:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2201,7 +2224,7 @@
           python_version: '3.11'
 
   j17_cqlsh_dtests_py38:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2219,7 +2242,7 @@
           python_version: '3.8'
 
   j17_cqlsh_dtests_py311:
-    <<: *j17_par_executor
+    <<: *j17_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2237,7 +2260,7 @@
           python_version: '3.11'
 
   j11_dtests_large_vnode:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2251,7 +2274,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2265,7 +2288,7 @@
           pytest_extra_args: '--only-resource-intensive-tests --force-resource-intensive-tests'
 
   j17_dtests_large_vnode:
-    <<: *j17_par_executor
+    <<: *j17_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2279,7 +2302,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j17_dtests_large:
-    <<: *j17_par_executor
+    <<: *j17_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
