--- config-2_1.yml	2023-01-19 06:40:13.405500420 -0600
+++ config-2_1.yml.MIDRES	2023-01-13 11:58:24.226316122 -0600
@@ -157,14 +157,14 @@
 j8_par_executor: &j8_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j8_small_par_executor: &j8_small_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j8_small_executor: &j8_small_executor
   executor:
@@ -172,29 +172,41 @@
     exec_resource_class: medium
   parallelism: 1
 
+j8_large_par_executor: &j8_large_par_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: large
+  parallelism: 50
+
+j8_very_large_par_executor: &j8_very_large_par_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: xlarge
+  parallelism: 100
+
 j8_medium_par_executor: &j8_medium_par_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 4
 
 j8_seq_executor: &j8_seq_executor
   executor:
     name: java8-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j11_par_executor: &j11_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_small_par_executor: &j11_small_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 10
 
 j11_small_executor: &j11_small_executor
   executor:
@@ -205,44 +217,56 @@
 j11_medium_par_executor: &j11_medium_par_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
-  parallelism: 1
+    exec_resource_class: large
+  parallelism: 4
+
+j11_large_par_executor: &j11_large_par_executor
+  executor:
+    name: java11-executor
+    exec_resource_class: large
+  parallelism: 50
 
 j11_seq_executor: &j11_seq_executor
   executor:
     name: java11-executor
-    #exec_resource_class: xlarge
+    exec_resource_class: medium
   parallelism: 1 # sequential, single container tests: no parallelism benefits
 
 j8_repeated_utest_executor: &j8_repeated_utest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j8_repeated_dtest_executor: &j8_repeated_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j8_repeated_upgrade_dtest_executor: &j8_repeated_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: xlarge
+  parallelism: 25
 
 j8_repeated_jvm_upgrade_dtest_executor: &j8_repeated_jvm_upgrade_dtest_executor
   executor:
     name: java8-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j11_repeated_utest_executor: &j11_repeated_utest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: medium
+  parallelism: 25
 
 j11_repeated_dtest_executor: &j11_repeated_dtest_executor
   executor:
     name: java11-executor
-  parallelism: 4
+    exec_resource_class: large
+  parallelism: 25
 
 j8_separate_jobs: &j8_separate_jobs
   jobs:
@@ -1837,7 +1861,7 @@
           target: testclasslist-system-keyspace-directory
 
   j8_dtests_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1851,7 +1875,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j8_dtests_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1865,7 +1889,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j11_dtests_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1880,7 +1904,7 @@
         pytest_extra_args: '--use-vnodes --num-tokens=16 --skip-resource-intensive-tests'
 
   j11_dtests_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1895,7 +1919,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --use-off-heap-memtables --skip-resource-intensive-tests'
 
   j8_dtests:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1909,7 +1933,7 @@
           pytest_extra_args: '--skip-resource-intensive-tests'
 
   j11_dtests:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
     - attach_workspace:
         at: /home/cassandra
@@ -1924,7 +1948,7 @@
         pytest_extra_args: '--skip-resource-intensive-tests'
 
   j8_upgrade_dtests:
-    <<: *j8_par_executor
+    <<: *j8_very_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1938,7 +1962,7 @@
           pytest_extra_args: '--execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all'
 
   j8_cqlsh_dtests_py3_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1953,7 +1977,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py3_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1968,7 +1992,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py38_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -1986,7 +2010,7 @@
           python_version: '3.8'
 
   j8_cqlsh_dtests_py311_vnode:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2004,7 +2028,7 @@
           python_version: '3.11'
 
   j8_cqlsh_dtests_py38_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2022,7 +2046,7 @@
           python_version: '3.8'
 
   j8_cqlsh_dtests_py311_offheap:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2040,7 +2064,7 @@
           python_version: '3.11'
 
   j8_cqlsh_dtests_py3:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2055,7 +2079,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j8_cqlsh_dtests_py38:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2073,7 +2097,7 @@
           python_version: '3.8'
 
   j8_cqlsh_dtests_py311:
-    <<: *j8_par_executor
+    <<: *j8_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2091,7 +2115,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py3_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2106,7 +2130,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py3_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2121,7 +2145,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2139,7 +2163,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_vnode:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2157,7 +2181,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py38_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2175,7 +2199,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311_offheap:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2193,7 +2217,7 @@
           python_version: '3.11'
 
   j11_cqlsh_dtests_py3:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2208,7 +2232,7 @@
           extra_env_args: 'CQLSH_PYTHON=/usr/bin/python3.6'
 
   j11_cqlsh_dtests_py38:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2226,7 +2250,7 @@
           python_version: '3.8'
 
   j11_cqlsh_dtests_py311:
-    <<: *j11_par_executor
+    <<: *j11_large_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2244,7 +2268,7 @@
           python_version: '3.11'
 
   j8_dtests_large_vnode:
-    <<: *j8_par_executor
+    <<: *j8_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2258,7 +2282,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j8_dtests_large:
-    <<: *j8_par_executor
+    <<: *j8_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2272,7 +2296,7 @@
           pytest_extra_args: '--only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large_vnode:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
@@ -2286,7 +2310,7 @@
           pytest_extra_args: '--use-vnodes --num-tokens=16 --only-resource-intensive-tests --force-resource-intensive-tests'
 
   j11_dtests_large:
-    <<: *j11_par_executor
+    <<: *j11_medium_par_executor
     steps:
       - attach_workspace:
           at: /home/cassandra
