--- a/bin/cassandra
+++ b/bin/cassandra
@@ -109,7 +109,7 @@
 fi
 
 if [ -z "$CASSANDRA_LOG_DIR" ]; then
-  CASSANDRA_LOG_DIR=$CASSANDRA_HOME/logs
+  CASSANDRA_LOG_DIR=/var/log/cassandra
 fi
 
 # Special-case path variables.
--- a/conf/cassandra-env.sh
+++ b/conf/cassandra-env.sh
@@ -93,16 +93,16 @@
     echo "$JVM_OPTS" | grep -q "^-[X]log:gc"
     if [ "$?" = "1" ] ; then # [X] to prevent ccm from replacing this line
         # only add -Xlog:gc if it's not mentioned in jvm-server.options file
-        mkdir -p ${CASSANDRA_HOME}/logs
-        JVM_OPTS="$JVM_OPTS -Xlog:gc=info,heap*=trace,age*=debug,safepoint=info,promotion*=trace:file=${CASSANDRA_HOME}/logs/gc.log:time,uptime,pid,tid,level:filecount=10,filesize=10485760"
+        mkdir -p /var/log/cassandra
+        JVM_OPTS="$JVM_OPTS -Xlog:gc=info,heap*=trace,age*=debug,safepoint=info,promotion*=trace:file=/var/log/cassandra/gc.log:time,uptime,pid,tid,level:filecount=10,filesize=10485760"
     fi
 else
     # Java 8
     echo "$JVM_OPTS" | grep -q "^-[X]loggc"
     if [ "$?" = "1" ] ; then # [X] to prevent ccm from replacing this line
         # only add -Xlog:gc if it's not mentioned in jvm-server.options file
-        mkdir -p ${CASSANDRA_HOME}/logs
-        JVM_OPTS="$JVM_OPTS -Xloggc:${CASSANDRA_HOME}/logs/gc.log"
+        mkdir -p /var/log/cassandra
+        JVM_OPTS="$JVM_OPTS -Xloggc:/var/log/cassandra/gc.log"
     fi
 fi
 
