diff --git a/src/java/org/apache/cassandra/config/DatabaseDescriptor.java b/src/java/org/apache/cassandra/config/DatabaseDescriptor.java
index 426544b..28a53f6 100644
--- a/src/java/org/apache/cassandra/config/DatabaseDescriptor.java
+++ b/src/java/org/apache/cassandra/config/DatabaseDescriptor.java
@@ -70,6 +70,9 @@ public class DatabaseDescriptor
     private static int replicationFactor_ = 3;
     private static long rpcTimeoutInMillis_ = 2000;
     private static Set<InetAddress> seeds_ = new HashSet<InetAddress>();
+    private static String zooKeeperRoot_ = "locks";
+    private static String zooKeeperPort_ = "2181";
+    private static Set<String> zooKeepers_ = new HashSet<String>();
     /* Keeps the list of data file directories */
     private static String[] dataFileDirectories_;
     /* Current index into the above list of directories */
@@ -579,6 +582,13 @@ public class DatabaseDescriptor
             {
                 seeds_.add(InetAddress.getByName(seeds[i]));
             }
+
+            /* zookeepers */
+            zooKeeperRoot_ = xmlUtils.getNodeValue("/Storage/ZooKeeperRoot");
+            zooKeeperPort_ = xmlUtils.getNodeValue("/Storage/ZooKeeperPort");
+            String[] zooKeepers = xmlUtils.getNodeValues("/Storage/ZooKeepers/ZooKeeper");
+            for (int i=0; i<zooKeepers.length; ++i)
+                zooKeepers_.add(zooKeepers[i]);
         }
         catch (ConfigurationException e)
         {
@@ -891,6 +901,21 @@ public class DatabaseDescriptor
         return seeds_;
     }
 
+    public static String getZooKeeperRoot()
+    {
+        return zooKeeperRoot_;
+    }
+
+    public static String getZooKeeperPort()
+    {
+        return zooKeeperPort_;
+    }
+
+    public static Set<String> getZooKeepers()
+    {
+        return zooKeepers_;
+    }
+
     public static String getColumnFamilyType(String tableName, String cfName)
     {
         assert tableName != null;
